latest/docker-entrypoint.sh

```diff
@@ -25,9 +25,9 @@ if [[ "$1" == "zeo"* ]]; then
   exec gosu senaite bin/$1 fg
 fi
 
-# Change the password
 if [[ -n "${PASSWORD}" ]]; then
-    gosu senaite bin/zope-passwd -p "${PASSWORD}"
```

Comment:
of course it would be better to get `zope-passwd` working again but unfortunately I have no idea how to do that :disappointed: